plugins/modules/win_domain_computer_info.ps1

```diff
@@ -0,0 +1,98 @@
+#!powershell
+Set-StrictMode -Version 2.0
+
+# Copyright: (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+#Requires -Module ActiveDirectory
+
+$spec = @{
+    options = @{
+        identity = @{ type = "str";  }
+        properties = @{type = "list"; elements = "str"; default = @('*')}
+        excluded_properties = @{type= "list"; elements = "str"; default = @('DeclaredConstructors', 'DeclaredMembers', 'nTSecurityDescriptor')}
+        search_scope = @{ type = 'str'; choices = @('Base', 'OneLevel', 'Subtree')}
```

Comment:
Have a look at `win_domain_object_info` https://github.com/ansible-collections/community.windows/blob/main/plugins/modules/win_domain_object_info.ps1#L20 to see how this is done there.